example/offload-template/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+#------------------------------- -*- cmake -*- -------------------------------#
+# Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+#-----------------------------------------------------------------------------#
+# Minimal Geant4 application with Celeritas offloading capabilities
+#-----------------------------------------------------------------------------#
+cmake_minimum_required(VERSION 3.10)
+cmake_policy(VERSION 3.12...3.30)
+project(offload-template)
+
+#-----------------------------------------------------------------------------#
+# Find packages
+find_package(Celeritas 0.5)
```

Comment:
Is the version 0.5 the minimum requirement?

Reply:
I think the current offload interface became available on 0.5, but that requirement is from my memory. I can double check it.

Reply:
There's no reason not to fix it at the current "minimum available" version: I think this is fine.